lib/src/main/java/com/orange/ods/compose/component/content/OdsComponentContent.kt

```diff
@@ -29,7 +29,7 @@ abstract class OdsComponentContent {
      * Calls `Content(Modifier)` with the default `Modifier`.
      */
     @Composable
-    fun Content() = Content(modifier = Modifier)
+    open fun Content() = Content(modifier = Modifier)
```

Comment:
I think we should keep this one final so that a subclass only has to override the other one and both work.

I originally wanted to write `open fun Content(modifier: Modifier = Modifier)` but using default values on abstract methods is not authorized (same for interface methods). That's why there are two methods.

For instance in `OdsButton` we only override `Content()` and if `Content(modifier)` is called it does nothing. If we override `Content(modifier)` only and keep `Content()` final, both versions of this method will work.